isis/src/base/apps/caminfo/caminfo.cpp

```diff
@@ -253,37 +277,15 @@ namespace Isis{
           CameraStatistics stats(filename, sinc, linc);
           Pvl camPvl = stats.toPvl();
 
-          PvlGroup cg = camPvl.findGroup("Latitude", Pvl::Traverse);
-          camstats->append(MakePair("MinimumLatitude", cg["latitudeminimum"][0]));
-          camstats->append(MakePair("MaximumLatitude", cg["latitudemaximum"][0]));
-
-          cg = camPvl.findGroup("Longitude", Pvl::Traverse);
-          camstats->append(MakePair("MinimumLongitude", cg["longitudeminimum"][0]));
-          camstats->append(MakePair("MaximumLongitude", cg["longitudemaximum"][0]));
-
-          cg = camPvl.findGroup("Resolution", Pvl::Traverse);
-          camstats->append(MakePair("MinimumResolution", cg["resolutionminimum"][0]));
-          camstats->append(MakePair("MaximumResolution", cg["resolutionmaximum"][0]));
-
-          cg = camPvl.findGroup("PhaseAngle", Pvl::Traverse);
-          camstats->append(MakePair("MinimumPhase", cg["phaseminimum"][0]));
-          camstats->append(MakePair("MaximumPhase", cg["phasemaximum"][0]));
-
-          cg = camPvl.findGroup("EmissionAngle", Pvl::Traverse);
-          camstats->append(MakePair("MinimumEmission", cg["emissionminimum"][0]));
-          camstats->append(MakePair("MaximumEmission", cg["emissionmaximum"][0]));
```

Comment:
I think these need to remain because removing them would cause breakage. These are the only keywords produced in the previous version. 

Tag for later removal like the typo?